### PR TITLE
Honor, fire, burn thy fear: Adds a simple animal knockout technique to the Kis-Khan martial art

### PR DESCRIPTION
**Pull request by feartheblackout**
Implements a disarm technique to Kis-Khan that "knocks out" simple animals for a few seconds, allowing you to follow up with other attacks. This is because shriekers are easy to encounter in the Untouched Lands as a Kis-Khan master and are too difficult to get away from with Unathi sprint, making it easy to get overwhelmed by multiple of them, especially if you're in water. And dying to shriekers is, unfortunately, not badass as a martial arts master.

Could also pave the way for other simple animal-based attacks for Kis-Khan or other martial arts, if people are interested in that.

### CODE REVIEW

**Comment by feartheblackout** [comment]
!review

**Review by DreamySkrell — APPROVED** [review]

File: code/modules/martial_arts/martial.dm [hunk 1]
```diff
@@ -163,6 +163,12 @@
 
 	return TRUE
 
+/datum/martial_art/proc/simple_animal_basic_disarm(var/mob/living/carbon/human/A, var/mob/living/simple_animal/D)
+	if(A.a_intent == I_DISARM)
+		A.visible_message("<b>\The [A]</b> [D.response_disarm] \the [D]")
```
Comment (FluffyGhoster) [inline code/modules/martial_arts/martial.dm:168]: SPAN_BOLD

File: code/modules/martial_arts/unathi.dm [hunk 2]
```diff
@@ -45,8 +45,22 @@
 	add_to_streak("D",D)
 	if(check_streak(A,D))
 		return 1
-	basic_hit(A,D)
-	return 1
+	if(istype(D, /mob/living/simple_animal))
+		simple_animal_basic_disarm(A,D)
+		return 1
```
Comment (FluffyGhoster) [inline code/modules/martial_arts/unathi.dm:50]: Condense into a single return at the end

File: code/modules/martial_arts/unathi.dm [hunk 3]
```diff
@@ -45,8 +45,22 @@
...
+	else
+		basic_hit(A,D)
+		return 1
```
Comment (FluffyGhoster) [inline code/modules/martial_arts/unathi.dm:53]: Condense into a single return at the end

File: code/modules/martial_arts/unathi.dm [hunk 4]
```diff
@@ -45,8 +45,22 @@
...
+
+/datum/martial_art/kis_khan/simple_animal_basic_disarm(var/mob/living/carbon/human/A, var/mob/living/simple_animal/D)
+	if(!D.paralysis)
+		A.visible_message("<b>[A]</b> delivers a blow to \the <b>[D]</b>'s head, making [D.get_pronoun("him")] fall unconscious!")
```
Comment (FluffyGhoster) [inline code/modules/martial_arts/unathi.dm:57]: SPAN_BOLD

File: code/modules/martial_arts/unathi.dm [hunk 5]
```diff
@@ -45,8 +45,22 @@
...
+		A.do_attack_animation(D)
+		playsound(D.loc, /singleton/sound_category/punch_sound, 25, 1, -1)
```
Comment (FluffyGhoster) [inline code/modules/martial_arts/unathi.dm:59]: This isn't an acceptable form of playsound, respect the header:

`playsound(get_turf(D), /singleton/sound_category/punch_sound, 25, TRUE, ONE_OF_THE_DEFINES_THAT_ENDS_WITH_SOUND_EXTRARANGE)`

(You can multiply/add/subtract from the extrarange define to fit)

File: code/modules/martial_arts/unathi.dm [hunk 6]
```diff
@@ -45,8 +45,22 @@
...
+		D.AdjustParalysis(5)
+	else
+		to_chat(A, SPAN_WARNING("\The <b>[D]</b> is already unconscious!"))
```
Comment (FluffyGhoster) [inline code/modules/martial_arts/unathi.dm:62]: SPAN_BOLD

**Review by FluffyGhoster — CHANGES_REQUESTED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Geevies — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**